app/controllers/foo_controller.rb

```diff
@@ -0,0 +1,13 @@
+class FooController < ActionController::Base  
+  def new_user
```

Comment:
__[staging][rails_best_practices]__
* move model logic into model (user use_count > 4) [:ref](http://rails-bestpractices.com/posts/2010/07/21/move-model-logic-into-the-model/)
* remove unused methods (FooController#new_user)